scripts/telemctl

```diff
@@ -18,6 +18,20 @@ declare -a SERVICES=(
 SCRIPT="$0"
 TELEM_DIR=/etc/telemetrics
 OPT_OUT_FILE=${TELEM_DIR}/opt-out
+TELEM_WRK_DIRS_CONF=/usr/lib/tmpfiles.d/telemetrics-dirs.conf
+
+create_work_dirs() {
+  # Creates dirs if missing, adjust ownership if exists
+  systemd-tmpfiles --create telemetrics-dirs.conf
+}
+
+telem_remove_work_dirs() {
+  # Remove dirs
+  while read LINE;
```

Comment:
How about: 

```
awk '{print $2}' $TELEM_WRK_DIRS_CONF | xargs rm -rf 
```

Reply:
vs. the whole while loop

Reply:
Or use that "read" can split the line for you and use

while read stuff dir junk
do rm -rf "$dir"
done < "$TELEM_WRK_DIRS_CONF"

We could even allow for the fact that you can have comments in the telemetrics-dirs.conf file, rather than having a problem with someone adding the following comment

`# / is getting full these days, so tidy up every now and then`

and then you doing "echo / | xargs rm -rf"

Reply:
I used a loop because I wanted to check for comments or empty lines. Since I did not add any checks I agree the loop is unnecessary.